ant/build.gant

```diff
@@ -1113,7 +1113,10 @@ junit = { pluginName ->
   compile("${pluginName}/test/junit", "build/test/junit/classes/${pluginName}"){
     ant.compilerarg(value: '-Xlint:-options')
     ant.include(name: '**/*.java')
-    ant.classpath{ant.pathelement(path: '${build.classes}/org.eclim')}
+    ant.classpath{
+      ant.pathelement(path: '${build.classes}/org.eclim')
+      ant.pathelement(path: '${build.classes}/org.eclim.jdt')
```

Comment:
This should be changed to to use `${pluginName}` rather than hardcoding in jdt, which not all users may be able to compile (they may only be interested in running/testing cdt for example).